projet/users/models.py

```diff
@@ -34,10 +35,10 @@ def _create_user(self, email, password, is_staff, is_superuser, **extra_fields):
         return user
 
     def create_user(self, email, password, **extra_fields):
-        return self._create_user(email, password, **extra_fields)
+        return self._create_user(email, password, False, False, **extra_fields)
 
     def create_superuser(self, email, password, **extra_fields):
-        user = self._create_user(self, email, password, **extra_fields)
+        user = self._create_user(email, password, True, True, **extra_fields)
```

Comment:
Yep. It's good you removed the self from where it was. Would have caused some problems.